Cargo.toml

```diff
@@ -42,7 +42,7 @@ generate-protos = ["tonic-build", "git2"]
 [dev-dependencies]
 structopt = "0.3"
 clap = "2.34"
-csv = "1.1"
+csv = "=1.1"
```

Comment:
This change was needed because csv 1.2 only supports rust 1.60 and up.

Reply:
It would be good to have a comment here then, so we can remove it when bumping the rust version in the future

Reply:
Added.